turbopack/crates/turbopack-cli/benches/small_apps.rs

```diff
@@ -27,7 +27,7 @@ fn list_apps() -> (PathBuf, Vec<PathBuf>) {
         let path = entry.path();
         if path.is_dir() {
             // Exclude node_modules
-            if path.file_name().unwrap().to_string_lossy() == "node_modules" {
+            if path.file_name().unwrap_or_default() == "node_modules" {
```

Comment:
i guess the alternative is something like `path.file_name() == Some(OsString::new("node_modules"))`, is that what ben was advocating for?  @bgw ?

Reply:
But `OsStr` implements `PartialEq<&str>`

Reply:
Yeah, that's what I was advocating, but I didn't realize about the `PartialEq` implementation. This looks nicer. Thanks.